LICENSE

```diff
@@ -1,21 +1,26 @@
-MIT License
+Copyright (c),(©) 2021 Sceleratis (Davey_Bones)
 
-Copyright (c) 2018 Sceleratis (Davey_Bones)
+This Software covered by Creative Commons license, available at
+"https://creativecommons.org/licenses/by/4.0/legalcode". 
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
+This is a human-readable summary of the license, and not
+a substitute for the license.
```

Comment:
```
This is a human-readable summary of the license, and not
a substitute for the license.
```
Please, please, please do your research. You can add a summary ABOVE the license, but not INSTEAD of it.